### PR TITLE
Added mirror view support

### PR DESCRIPTION
**Pull request by smronju**
User will be able to choose view they prefer.

### CODE REVIEW

**Comment by mozmorris** [comment]
@smronju just picking up on this, sorry for the delay. Thanks for the PR!

The PR itself has modified the .gitignore file and subsequently included the build files. We'd need to revert those changes.

I actually think this is quite a useful feature and it is something I'd like to add. If you could update the PR, we could look to get this merged.

**Comment by smronju** [comment]
Okay just removed the build files and updated .gitignore.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ for browser compatibility.
 ## Installation
 
 ```
-npm install react-webcam
+npm install git+https://github.com/smronju/react-webcam.git
```
Comment (mozmorris) [inline README.md]: This needs reverting.

File: src/react-webcam.js [hunk 2]
```diff
@@ -183,7 +185,15 @@ export default class Webcam extends Component {
       canvas.height = video.clientHeight;
       canvas.width = video.clientWidth;
       this.canvas = canvas;
-      this.ctx = canvas.getContext('2d');
+
+      if (this.props.mirror) {
```
Comment (mozmorris) [inline src/react-webcam.js]: Can this be refactored?

e.g
```javascript
this.ctx = canvas.getContext('2d');

if (this.props.mirror) {
  this.ctx.translate(canvas.width, 0);
  this.ctx.scale(-1, 1);
}
```

File: src/react-webcam.js [hunk 3]
```diff
@@ -193,13 +203,24 @@ export default class Webcam extends Component {
...
+    };
+
     return (
       <video
         autoPlay
         width={this.props.width}
         height={this.props.height}
         src={this.state.src}
         className={this.props.className}
+        style={(this.props.mirror) ? style.mirror : null}
```
Comment (mozmorris) [inline src/react-webcam.js]: Nit: The extra parenthesis can be removed. 

```javascript
style={this.props.mirror ? style.mirror : null}
```

**Pull request closed without merging**